README.md

```diff
@@ -194,6 +200,23 @@ Install FFTW from ports tree or ``pkg``:
 
 Please install all of them, if possible.
 
+## Testing
+
+Tests should be run using `pytest`. Install using:
```

Comment:
Is this necessary? Does `python -m unittest discover` not work any more?

Reply:
One could possibly continue to just use `unittest`, but for CI I use `pytest` because it allows us to specifically target the generated `wheel` over the local (uncompiled) python module.

I could not get `unittest` discoverer to easily place the installed wheel folder before the local directory in `sys.path`.

No changes to the workflow have been made, so `python -m unittest discover` will continue to work as before.

Reply:
Edit: this is not entirely true. The folder with the tests have been renamed from `test` to `tests` to avoid a namespace conflict with the built-in `test` module from CPython.

I assume `unittest` will also discover the test directory when it is named `tests`. I believe it is recommended for it to be named `tests` anyway for the reason mentioned above. This change was necessary to ensure the proper module resolution.

Reply:
Feel free to change this if you prefer the other method to be the documented approach. The underlying tests remain the same - the only difference is which test runner is used. `pytest` arguable is more easily configurable and is a common choice in my experience.